server/api/utils/clients/iguazio.py

```diff
@@ -610,7 +618,7 @@ def _verify_job_in_terminal_state():
 
         job_state, job_result = mlrun.utils.helpers.retry_until_successful(
             self._wait_for_job_completion_retry_interval,
-            360,
+            timeout,
```

Comment:
config timeout is a string
```suggestion
            int(timeout),
```